nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Updater/PackageReferenceUpdater.cs

```diff
@@ -673,11 +695,18 @@ private static IEnumerable<IXmlElementSyntax> FindPackageNodes(
         ProjectBuildFile buildFile,
         string packageName)
         => buildFile.PackageItemNodes.Where(e =>
-            string.Equals(
-                (e.GetAttributeOrSubElementValue("Include", StringComparison.OrdinalIgnoreCase) ?? e.GetAttributeOrSubElementValue("Update", StringComparison.OrdinalIgnoreCase))?.Trim(),
-                packageName,
-                StringComparison.OrdinalIgnoreCase) &&
-            (e.GetAttributeOrSubElementValue("Version", StringComparison.OrdinalIgnoreCase) ?? e.GetAttributeOrSubElementValue("VersionOverride", StringComparison.OrdinalIgnoreCase)) is not null);
+        {
+            // Attempt to get "Include" or "Update" attribute values
+            var includeOrUpdateValue = e.GetAttributeOrSubElementValue("Include", StringComparison.OrdinalIgnoreCase)
+                                    ?? e.GetAttributeOrSubElementValue("Update", StringComparison.OrdinalIgnoreCase);
+            // Trim and split if there's a valid value
+            var packageNames = includeOrUpdateValue?.Trim().Split(';')
+                                .Where(t => t.Equals(packageName, StringComparison.OrdinalIgnoreCase));
```

Comment:
You'll need another `.Trim()` call here; it's perfectly valid to have `Include=" Package1 ; Package2 "`.